Tribler/Test/test_gui_dialogs.py

```diff
@@ -176,18 +175,26 @@ def disable_dispersy():
         self.startTest(disable_dispersy)
 
     def test_confirmationdialog(self):
```

Comment:
@whirm I notice that `test_remove_dialog` has the same structure as the confirmationdialog test. Should I also change it into the structure that I have now?